src/components/organisms/CompareModal/CompareModal.tsx

```diff
@@ -36,25 +37,36 @@ export const DiffModal: React.FC<Props> = ({visible, onClose}) => {
       {!diffComparison ? <CompareActionBar /> : <DiffActionBar />}
 
       <Row ref={containerRef}>
-        <Col span={11}>
+        <Col span={10}>
           <ResourceSetSelector side="left" />
         </Col>
-        <Col span={2} />
-        <Col span={11}>
+        <Col span={4} />
+        <Col span={10}>
           <ResourceSetSelector side="right" />
         </Col>
       </Row>
 
       <div
         style={{
```

Comment:
Can we have a styled component for this <div> ? Also why it is needed to use `calc` for the height?

Reply:
I still want to focus a bit on functionality before getting code style in order as things are still in flux a lot. I promise I'll extract this before its done.

As for the `calc`, It's needed because the resource set selector can increase in height but I need fixed height on this column to get proper overflow behaviour.